MonstieWash/Assets/Scripts/UI/MainMenu.cs

```diff
@@ -1,24 +1,23 @@
 using System.Collections;
 using System.Collections.Generic;
 using UnityEngine;
-using UnityEngine.UI;
 
 public class MainMenu : MonoBehaviour
 {
-    [SerializeField] private Button startButton;
     private GameSceneManager m_gameSceneManager;
+    [SerializeField] private Canvas m_menuCanvas;
```

Comment:
What's up with this line? I can't see it used anywhere for anything. If it's from an old test or smthn and it's not being used for anything then pls just remove it.